src/modals/confirmation-modal/operations-confirmation/operations-preview/operations-preview-item/operations-preview-item.tsx

```diff
@@ -117,13 +117,15 @@ export const OperationsPreviewItem: FC<Props> = ({ paramPreview }) => {
                   showMinusSign
                 />
                 <Divider size={formatSize(8)} />
```

Comment:
do we need to show divider for collectible?

Reply:
yes, it does padding from the bottom